### ISSUE TITLE
SDL2 clients work poorly

### ISSUE THREAD

**Issue by Unesty** [st3r4g/swvkc#5]
They can work by setting wl_data_device_manager global to 3. They have tearing and low fps. Also not fullscreen SDL2 clients buffers stay on screen after exit.

**Comment by st3r4g** [comment]
Yes that version should be set to 3. Which SDL2 client specifically? I'm trying with `mgba` and it seems to run fine (no tearing, 60 fps). The buffers are staying on screen after exit because I'm not doing any cleanup at the moment, I'll fix it soon.

**Comment by Unesty** [comment]
mgba run with tearing and cracking sound because of snd_hda_intel position_fix=1 as well as unity engine games in Xorg, probably because of high CPU load.

**Comment by Unesty** [comment]
I used mercurial version of SDL2.
